mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -17,11 +18,22 @@ export const UserPage = () => {
         level={HeadingLevel.h2}
         text="User page"
       />
-      <Title
-        text="Own ways"
-        level={HeadingLevel.h3}
-      />
-      {uuid ? <OwnWaysTable uuid={uuid} /> : <Navigate to={pages.page404.path} />}
+      {uuid
+        ? (
+          <>
+            <Title
+              text="Own ways"
+              level={HeadingLevel.h3}
+            />
+            <OwnWaysTable uuid={uuid} />
+            <Title
+              text="Mentoring ways"
+              level={HeadingLevel.h3}
+            />
+            <MentoringWaysTable uuid={uuid} />
+          </>
+        )
```

Comment:
I think better move this logic to separate function

Reply:
I think it's ok right now, another function is kinda unnecessary in my opinion

Reply:
Personally I don't like ternarian operator is such cases. Also I think that we should not redirect from jsx, better to redirect on the next line after reading  params, it is about encapsulation, let's keep cohesive data and logic nearby